app/AppKernel.php

```diff
@@ -21,6 +21,7 @@ public function registerBundles()
             new Doctrine\Bundle\DoctrineBundle\DoctrineBundle(),
             new Sensio\Bundle\FrameworkExtraBundle\SensioFrameworkExtraBundle(),
             new AppBundle\AppBundle(),
+            new Knp\Bundle\PaginatorBundle\KnpPaginatorBundle()
```

Comment:
this bundle should be registered before AppBundle. It generally makes sense to keep AppBundle at the end of the list